app/search/routes.py

```diff
@@ -131,7 +127,7 @@ def dataset_search():
     limit = max(min(int(request.args.get('limit') or 10), 10), 0)
     sort_key = request.args.get('sortKey') or "id"
     paginated = elements[(cursor):(cursor + limit)]
-    paginated.sort(key=lambda o: o[sort_key])
+    paginated.sort(key=lambda o: (o[sort_key] is None, o[sort_key]))
```

Comment:
See https://stackoverflow.com/questions/18411560/python-sort-list-with-none-at-the-end